.github/actions/upload-coverage/action.yml

```diff
@@ -0,0 +1,19 @@
+name: Upload Coverage
+description: Upload coverage to codecov
+
+inputs:
+  name:
+    description: "Job name"
+    required: true
+
+runs:
+  using: "composite"
+
+  steps:
+    - run: |
+        cd mfem
+        curl -o codecov.sh -f https://codecov.io/bash || \
+          curl -o codecov.sh -f https://codecov.io/bash || \
+          curl -o codecov.sh -f https://codecov.io/bash
```

Comment:
I don't really understand this

Reply:
I admit I took it from another project: I understand it as a triple attempt to download the script, in case latency results into a failure.
Since the project I took it from is quite serious with code coverage, I thought it might make sense.

Reply:
hack level: pro